client/src/features/DataTable.js

```diff
@@ -76,17 +76,17 @@ const HeaderRow = ({ onSelectAllClick, numSelected, rowCount, data }) => {
   );
 }
 
-const DataTable = ({data, func}) => {
+const DataTable = (props) => {
```

Comment:
It seems to be more logical to have the props instead of destructuring for its reusability!

Reply:
I actually just changed that because it wasn’t working in my branch and I was trying to get it to work. When Yang merges his it’ll be back the original way. I think I like to not destructure the props though so it’s clearer in the rest of the code what variables are part of the props.